From: Thomas Renninger <trenn@suse.de>
Subject: Avoid early hang when i8042 controller is missing
Patch-Mainline: no
References: bnc#528811

On latest machines without i8042 controller LED control
will hang when BIOS option "PORT 60/64 emulation" is enabled
(which is required for specific extra storage BIOS to work).

Workaround for now: Don't touch LEDs at all.

---
 arch/x86/kdb/kdba_io.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-2.6.32-master/arch/x86/kdb/kdba_io.c
===================================================================
--- linux-2.6.32-master.orig/arch/x86/kdb/kdba_io.c
+++ linux-2.6.32-master/arch/x86/kdb/kdba_io.c
@@ -23,7 +23,7 @@
 #include "pc_keyb.h"
 
 #ifdef	CONFIG_VT_CONSOLE
-#define KDB_BLINK_LED 1
+#undef	KDB_BLINK_LED
 #else
 #undef	KDB_BLINK_LED
 #endif
